codegen/smithy-aws-swift-codegen/src/test/resources/software.amazon.smithy.aws.swift.codegen/route53-invalidbatch.smithy

```diff
@@ -4,7 +4,7 @@ namespace com.amazonaws.route53
 use aws.api#service
 use aws.protocols#restXml
 
-@service(sdkId: "Route53")
+@service(sdkId: "Route 53")
```

Comment:
`Route 53` (with space) is the actual sdkId for Route 53.

Reply:
This whole Route 53 customization is not tested, that will happen with this ticket:
https://github.com/awslabs/aws-sdk-swift/issues/1027